tests/unittests/evaluation_tests.py

```
@@ -242,6 +256,80 @@ def test_regression_custom_metric(self):
+        self.assertTrue(dataset._has_sidebar_group("tmp"))
+        self.assertFalse(dataset._has_sidebar_group("still_eval"))
+
+        dataset.app_config.sidebar_groups = None
+        dataset.save()
+
+        # Existing sidebar group of same name
+
+        dataset._add_paths_to_sidebar_group(["id"], "eval")
+        dataset._add_paths_to_sidebar_group(["filepath"], "still_eval")
+
+        results = dataset.evaluate_regressions(
```

Ruff (0.8.2)

Local variable `results` is assigned to but never used

Remove assignment to unused variable `results`

(F841)